cub/cub/device/dispatch/dispatch_segmented_sort.cuh

```diff
@@ -29,6 +29,8 @@
 
 #include <cub/config.cuh>
 
+#include <atomic>
+
```

Comment:
Q: What is this include needed for?

Reply:
I don't know how that made its way in... I removed it.